tool/autoqa/schemaorg/trim-class.js

```diff
@@ -172,6 +175,8 @@ class SchemaTrimmer {
 
         for (let fieldname in type.fields) {
             const field = type.fields[fieldname];
+            if (this._whiteListed(`${arg.name}.${fieldname}`))
```

Comment:
Is there a case here where you can have two level nesting of compound types?

Reply:
fixed